src/provideEndpoints.ts

```diff
@@ -0,0 +1,89 @@
+import { Endpoints } from './models/core/Endpoints';
+
+export const defaultApiVersion = 20220511;
+
+/**
+ * Defines the cloud region of the API domains.
+ *
+ * @public
+ */
+export enum CloudRegion {
+  US = 'us',
+  EU = 'eu',
+}
+
+/**
+ * Defines the environment of the API domains.
+ *
+ * @public
+ */
+export enum Environment {
+  PROD = 'prod',
+  SANDBOX = 'sbx',
+}
+
+/**
+ * Provides methods for getting various endpoints.
+ *
+ * @internal
+ */
+export class EndpointsProvider {
```

Comment:
sorry my only other thought is that we might want to not call this a Provider since it is not a React Context Provider

Reply:
Done